.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,35 @@
+---
+name: Bug report
+about: Found a bug? Let us know.
+title: ''
+labels: [bug]
+assignees: ''
+---
+
+**What bug did you find?**
+A clear and concise description of the bug.
+
+**How to reproduce the bug**
+Please provide (if available):
+- WarpX inputs files
+- PICMI Python files
+- Python post-processing scripts
+
+Please minimize your inputs/scripts to be concise and focused on the issue.
+For instance, make the simulation scripts as small and fast to run as possible.
+
+**What behavior did you expect?**
+A clear and concise description of what you expected to happen.
+
+**Tell us about your software environment**
+- operating system: [name and version]
+- version of WarpX: [e.g., latest, 24.09, etc.]
+- installed WarpX via: [e.g., conda-forge, spack, pip, brew, from source, module system, etc.]
+- HPC machine: [Are you running on an HPC cluster? We might support it already: https://warpx.readthedocs.io/en/latest/install/hpc.html]
+- other relevant dependency names and versions
+
```

Comment:
Should we add something like:

Have you tried compiling and running in debug mode?
If so, attach one of the Backtraces files. 

I feel like this is also a recurring step that could be skipped by adding it here.